model/src/commonMain/kotlin/SessionType.kt

```diff
@@ -7,7 +7,7 @@ enum class SessionType(val id: String, val isFavoritable: Boolean) {
     Codelabs("codelabs", true),
     FiresideChat("fireside_chat", false),
     Lunch("lunch", false),
-    Break("break", false),
+    BreakTime("break", false),
```

Comment:
Kotlin object named `Break` converted into "break" at Framework's header files.  
Header files are C's headers, so it causes compile error because `break` is keyword of C.